blackbox/docs/configuration.txt

```diff
@@ -1851,6 +1851,25 @@ to your config. This will prevent allocating primary and replica
 of the same shard on the same machine even if more than one instances
 running on it.
 
+Other System Configuration
+==========================
+
+Virtual Memory
+--------------
+
+CrateDB uses a hybrid ``mmapfs``/``niofs`` directory by default to store its
+indices. The defaultoperating system limits on ``mmap`` counts is likely to be
+too low, which may result in out of memory exceptions.
```

Comment:
I'm not entirely sure which exception this would specifically create - ES just says it like here. Could also mention the bootstrap check here, too? Setting this value too low results in:

```
[WARN ][o.e.b.BootstrapCheck     ] [Hochstuhl] max virtual memory areas vm.max_map_count [65530] is too low, increase to at least [262144]
```